src/domain/application/applicationView/hooks/useSendTaydennys.tsx

```diff
@@ -78,13 +98,39 @@ export default function useSendTaydennys(
   const sendTaydennysDialog = (
     <ConfirmationDialog
       title={t('taydennys:sendDialog:title')}
-      description={t('taydennys:sendDialog:description')}
+      description={
+        isError ? (
```

Comment:
`sendTaydennysMutation` has `isError` property which could be used here so there would be no need for separate `isError` variable.

Reply:
Thank you for noticing! Changed it.

Reply:
Although, after this change, when there is an error in sending and the user closed the dialog after the error, then when opening the send dialog again the error message is shown right away. How can I reset mutation.isError so that when re-opening the dialog the error is not shown right away?

Reply:
Found it. Mutation has reset() function that should be called when closing the dialog.

Reply:
And it seems that I could also get rid off [isLoading, setIsLoading] and replace it with mutation.isLoading?

Reply:
Yes, seems like a good change!